acceptance/schema/schema.rb

```diff
@@ -7,84 +7,86 @@
 # frozen_string_literal: true
 
 ActiveRecord::Schema.define do
-  create_table :firms do |t|
-    t.string  :name
-    t.integer :rating
-    t.string :description
-    t.references :account
-  end
+  ActiveRecord::Base.connection.ddl_batch do
```

Comment:
This ensures the test schema is created as one DDL batch, which speeds up the creation of the test database on real Cloud Spanner significantly. The execution speed difference on the emulator is negligible.